shell/common/animator.cc

```diff
@@ -41,6 +41,12 @@ void Animator::Start() {
   RequestFrame();
 }
 
+// This ID is used by the timeline component to correctly align
+// GPU Workloads events with their respective Framework Workload.
+const char* Animator::FrameId() {
```

Comment:
Why is this alternating instead of increasing?

Reply:
This values is used has a helper by the Frame alignment code to correctly pair `Framework Workload` events with the relative `GPU Workload`.
https://github.com/dart-lang/sdk/blob/master/runtime/observatory/web/timeline.js#L284

Due the fact that we can add just strings to the Timeline Events, alternating two constant values avoids us to print the value to it.

Reply:
Maybe FrameParity instead then, since this doesn't identify a unique frame.

Reply:
done